pkg/plugins/mongodb-atlas/cmd/main/main.go

```diff
@@ -155,15 +155,10 @@ func (a *AtlasCostSource) GetCustomCosts(req *pb.CustomCostRequest) []*pb.Custom
 		}
 
 		log.Debugf("fetching atlas costs for window %v", target)
-		result, err := a.getAtlasCostsForWindow(&target, lineItems)
-		if err != nil {
-			log.Errorf("error getting costs for window %v: %v", target, err)
-			errResp := pb.CustomCostResponse{}
-			errResp.Errors = append(errResp.Errors, fmt.Sprintf("error getting costs for window %v: %v", target, err))
-			results = append(results, &errResp)
-		} else {
-			results = append(results, result)
-		}
+		result := a.getAtlasCostsForWindow(&target, lineItems)
```

Comment:
getAtlasCostsForWindow was always returning a nil error. So no need to test for it. can make calling code simpler.